core/src/toga/screen.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from toga.images import Image
+from toga.platform import get_platform_factory
+
+
+class Screen:
+    def __init__(self, _impl):
+        self._impl = _impl
+        self.factory = get_platform_factory()
+
+    @property
+    def name(self) -> str:
+        """Unique name of the screen."""
+        return self._impl.get_name()
+
+    @property
+    def origin(self) -> tuple[int, int]:
+        """The absolute coordinates of the screen's origin, as a ``(x, y)`` tuple."""
+        return self._impl.get_origin()
+
+    @property
+    def size(self) -> tuple[int, int]:
+        """The size of the screen, as a ``(width, height)`` tuple."""
+        return self._impl.get_size()
+
+    def as_image(self, format=Image) -> Image:
```

Comment:
This type annotation isn't correct - compare with the type annotation of `ImageView.as_image()`.
```suggestion
    def as_image(self, format: type[ImageT] = Image) -> ImageT:
```
This will also require the import of ImageT from `toga.images` (again; cf ImageView).

Reply:
Done. Thanks.